examples/quickstart_etl/README.md

```diff
@@ -0,0 +1,247 @@
+
+## Step 3: Scheduling a daily job
+
+Finally, let's refresh our plots every day, so we can monitor the popular topics over time. To do so, we can use [Schedules](https://docs.dagster.io/concepts/partitions-schedules-sensors/schedules#schedules).
+
+We've defined a daily schedule in [quickstart_etl/repository.py](./quickstart_etl/repository.py) (which is Dagster's word for "project") for all assets that are defined in the [quickstart_etl/assets/](./quickstart_etl/assets) module.
+
+Now, let's set up the daily job with Dagster.
+
+In the UI, clicking "all_assets_job" in the left nav will bring you to the job definition page. In the left nav, it indicates the `all_assets_job` has a schedule associated with it; on the job page, you will find more info about the schedule.
+
+<p align="center">
+    <img height="500" src="../../docs/next/public/images/quickstarts/basic/step-3-1-schedule-off.png" />
+</p>
+
+ You can simply turn on the schedule switch to set up the daily job we defined in [quickstart_etl/repository.py](./quickstart_etl/repository.py).
+
+<p align="center">
+    <img height="500" src="../../docs/next/public/images/quickstarts/basic/step-3-2-schedule-on.png" />
+</p>
+
+### (Optional) Running daemon locally
+
+If you're running Dagster locally, you will see a warning that your daemon isn’t running.
+
+<p align="center">
+    <img height="300" src="../../docs/next/public/images/quickstarts/basic/step-3-3-daemon-warning.png" />
+</p>
+
+<details><summary>👈 Expand to learn how to set up a local daemon</summary>
+
+If you want to enable Dagster [Schedules](https://docs.dagster.io/concepts/partitions-schedules-sensors/schedules) for your jobs, start the [Dagster Daemon](https://docs.dagster.io/deployment/dagster-daemon) process in the same folder as your `workspace.yaml` file, but in a different shell or terminal.
+
+The `$DAGSTER_HOME` environment variable must be set to a directory for the daemon to work. Note: using directories within /tmp may cause issues. See [Dagster Instance default local behavior](https://docs.dagster.io/deployment/dagster-instance#default-local-behavior) for more details.
+
+In this case, go to the project root directory and run:
+```bash
+dagster-daemon run
+```
+
+Once your Dagster Daemon is running, the schedules that are turned on will start running.
+
+<p align="center">
+    <img height="500" src="../../docs/next/public/images/quickstarts/basic/step-3-4-daemon-on.png" />
+</p>
+
+</details>
+
+<br />
+<br />
+
+Congratulations 🎉 You now have a daily job running in production!
+
+---
+## Learning more
+
+### Changing the code locally
+
+When developing pipelines locally with the UI, be sure to click the "Reload definitions" button when you change the code. This ensures that the UI picks up the latest changes you made.
+
+You can reload the code in the "Deployment" page.
```

Comment:
```suggestion
You can reload the code using the "Deployment" page:
```

Should we make page names bold as well? In some places, page names don't have quotes but it does here.